### ISSUE TITLE
A server error occurred.

### ISSUE THREAD

**Issue by shawn1thompson** [39aldo39/Radicale-DecSync#23]
I have a fresh install of radicale which was working fine until I added the decsync plugin. Now, I'm getting an exception occurred du> when running `journalctl --unit radicale.service`. I don't know how to read the rest of the error.

**Comment by 39aldo39** [comment]
Radicale will generally just log to stdout. I am not sure why it doesn't show it with `journalctl`, but maybe you could try executing `python3 -m radicale` manually to debug the error?

**Comment by shawn1thompson** [comment]
Using `python3 -m radicale`, I received the following:

`python3 -m radicale
[2022-05-20 09:02:43 -0500] [53870/Thread-1 (process_request_thread)] [ERROR] An exception occurred during GET request on '/': [Errno 13] Permission denied: '/var/lib/radicale/collections/.Radicale.lock'`

I checked the permissions on the collections folder as well as its contents and they all belong to the radicale user and group. That error might be specific to the way I just ran it. It's normally started as a service using systemctl.
